Library/Formula/asciidoc.rb

```diff
@@ -17,6 +17,7 @@ class Asciidoc < Formula
   end
 
   depends_on "docbook"
+  option "with-docbook-xsl", "Install DTDs to generate manpages"
 
```

Comment:
Right. Sorry, I misunderstood the documentation. I'll push an updated branch shortly.